Loop/Managers/NightscoutDataManager.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  NightscoutDataManager.swift
+//  Loop
+//
+//  Created by Nate Racklyeft on 8/8/16.
+//  Copyright © 2016 Nathan Racklyeft. All rights reserved.
+//
+
+import Foundation
+import NightscoutUploadKit
+
+
+class NightscoutDataManager {
+
+    unowned let deviceDataManager: DeviceDataManager
+
+    init(deviceDataManager: DeviceDataManager) {
+        self.deviceDataManager = deviceDataManager
+
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(uploadLoopStatus(_:)), name: LoopDataManager.LoopDataUpdatedNotification, object: deviceDataManager.loopManager)
+    }
+
+    @objc func uploadLoopStatus(note: NSNotification) {
+
```

Comment:
`LoopDataUpdatedNotification` is posted when any data changes, so it probably needs to include some context in `userInfo` as to what changed since (I believe) you're only interested in posting data after a Loop attempt.